README.md

```diff
@@ -31,31 +31,34 @@ class HelloPage extends React.Component {
 
 }
 
-// Your route object should contain at least:
-// - The name of the route (which will become the navigation bar title)
-// - The component object for the page to render
-var firstRoute = {
-  name: 'Welcome!',
-  component: HelloPage
-};
-
 // The Router wrapper
 class MyApp extends React.Component {
 
   constructor(props) {
     super(props);
+
     this.styles = StyleSheet.create({
       header: {
         backgroundColor: '#5cafec',
       },
     });
+
+    // Your route object should contain at least:
+    // - The name of the route (which will become the navigation bar title)
+    // - The component object for the page to render
+    this.firstRoute = {
+      name: 'Welcome!',
+      component: HelloPage,
+    }
```

Comment:
why attach everything to the class? what's the advantage over just using `var` outside of the constructor?